docs/services-developer-guide.md

```diff
@@ -62,6 +59,16 @@ These resources are powered by:
    controller watches for changes to `CloudMemorystoreInstance` resources and
    calls Google's Cloud Memorystore API to create, update, or delete an instance
    as necessary.
+1. The resource claim scheduling controller. A claim scheduling controller
+   exists for each kind of resource class that could satisfy a resource claim.
+   This controller is unaware of any external system - it simply schedules
+   resource claims to resource classes that match their class selector labels,
+   so that they may be handled by the resource claim controller.
```

Comment:
Would it be appropriate to add here that if multiple classes exist that match labels then the controller will select at random?

Reply:
I thought about that, but I don't think going into detail on how these controllers work from a user's perspective is best explained in the developer guide. I do think the developers should be familiar with these concepts, but it's probably better to clarify things like this in (e.g) the services guide and consider knowing how to use services a prerequisite to building new services.